src/pages/Info/Info.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { MetaData } from '../../components/Meta/MetaData';
+import infoPDF from '../../assets/info.pdf';
+
+const meta = {
+	title: 'Info',
+	desc: 'CougarCS Information Packet',
+	url: 'https://cougarcs.com/info',
+	img: 'https://i.ibb.co/NTLFrdj/cougarcs-background11.jpg',
+};
+
+const Info = () => {
+	return (
+		<div>
```

Comment:
Can u make this div full view height of the page?

Reply:
Hmm how do you mean? I kinda like how @LeftAutomated includes the website nav at the top of the page. It lets people we directed to the PDF hop right into our website.

![image](https://user-images.githubusercontent.com/29151058/133897511-422ee640-f329-475d-9e36-70dc0e748d3b.png)

However, I can absolutely see the point that maybe we shouldn't include the nav and just display the PDF full screen. I could go either way, what do you guys think? @vyas0189 @KevFlo @shubhsheth @LeftAutomated

Reply:
![image](https://user-images.githubusercontent.com/25915724/133897608-44ee17e8-7671-46ce-8a1f-ddbca160e404.png)

Reply:
His current shows the footer in the page view

Reply:
To be honest, I do like how this looks without the footer, it looks more streamlined. While with the footer it looks more professional/ structured if that makes any sense.  Maybe if we can make the footer on this page alone like 25% smaller i'd be 
 game.